docs/managing-dragonfly/flags.md

```diff
@@ -0,0 +1,659 @@
+# Flags
+
+Dragonfly can be tuned and configured by a set of config flags. These flags can be:
+
+1. Passed to a Dragonfly instance directly at startup. E.g, `./dragonfly --port=6379`.
+2. Loaded from a file by a Dragonfly instance. E.g. `./dragonfly --flags-file=path_to_flags/flags.txt`
+3. Can be set via env variables. E.g. `export DFLY_PASSWORD=some_password`
```

Comment:
Here make sure to explain the case sensitivity of flags. Specifically `PASSWORD` is not a flag, but just an env variable.
I'd use `DFLY_port` to both demonstrate case sensitivity and be aligned with your above example

Reply:
I agree. DFLY_PASSWORD is actually a bad example since it's deprecated. I would even remove this support from the code for 1.13.